### ISSUE TITLE
Issues when taking pictures in series mode

### ISSUE THREAD

**Issue by robertnatter** [TUWien/DocScan#17]
When taking pictures in series mode DocScan sometimes wrongly recognises pages as already scanned, the problem can be "solved" by slightly moving the document/book but is still rather cumbersome especially when sitting in front of the ScanTent without seeing your phonescreen.

A second problem is, that sometimes pictures are taken although pages (or hands) are still moving, creating blurry images and often leading to problem number one.

Problem three happens rarely but sometimes, if a page is not turned fast enough a second picture of it is taken, basically meaning that the page was not recognised as already scanned.

Phone used: LG G2

Best wishes,
Robert

**Comment by hollaus** [comment]
Thank you for report! I agree that the series mode should be improved. This improvement will be implemented in a future release, but it will take some time to get there. I will post in this thread once the improvement is implemented.

Greetings
Fabian

**Comment by skurzinz** [comment]
Corroborating robertnatter's report, using a Motorola Moto G5 for a first proper test with ScanTent. 
All three issues popped up when going through a 40 page/20 image document. 

Issue 1: Source spread had empty page on the left (one-sided document), and there was little change on the right page as compared to the last pic (a long table with single rows continuing over 3 pages). It was hard to convince DocScan that this is indeed not the same page. Even moving the pages apart did not help at first. Just an idea without knowing how the recognition works: If documents are very similar on the first glimpse/calculation, look into smaller quadrants to check on changes? 

Issue 2: I had to delete 3 images unintentionally taken, some even with hands in them.  
Issue 3 (which I don't know is an issue separate from number 2) appeared only once, resulting in one of the hand-pics. Perhaps I stopped moving my hand for a moment.
I'll continue testing.
